spec/index.bs

```diff
@@ -192,34 +192,82 @@ dictionary GPULimits {
 </script>
 
 
-Buffers {#buffers}
-==================
+# {{GPUBuffer}} # {#GPUBuffer}
 
-## GPUBuffer ## {#buffer}
+A {{GPUBuffer}} represents a block of memory that can be used in GPU operations.
+Data is stored in linear layout, meaning that each byte of the allocation can be
+addressed by its offset from the start of the {{GPUBuffer}}, subject to alignment
+restrictions depending on the operation..
 
-<script type=idl>
-interface GPUBuffer : GPUObjectBase {
-    Promise<ArrayBuffer> mapReadAsync();
-    Promise<ArrayBuffer> mapWriteAsync();
-    void unmap();
+{{GPUBuffer}} has the following internal slots:
 
-    void destroy();
-};
-</script>
+<dl dfn-type=attribute dfn-for="GPUBuffer">
+	: <dfn>\[[size]]</dfn>
+	::
+        The Length of the {{GPUBuffer}} allocation in bytes, which is an unsigned long long.
+
+	: <dfn>\[[usage]]</dfn>
+	::
+        The allowed usages for this {{GPUBuffer}}, represented as a bitset unsigned long.
+
+    : <dfn>\[[state]]</dfn>
```

Comment:
nit: probably good to make a separate `<dfn dfn>buffer state</dfn>` which defines these 3 possible values (in prose), and use that as the "of type" here. Quotes around the state names probably isn't necessary, but they should also be `<dfn dfn>`s for cross-referencing.

Reply:
Figured out how to do this: `<dfn dfn>buffer state</dfn> ... <dfn dfn for="buffer state">mapped</dfn>`